app/_includes/docs-sidebar.html

```diff
@@ -1,5 +1,47 @@
 <aside class="docs-sidebar">
   <i class="fa fa-times close-sidebar"></i>
+
+  <div class="sidebar-title-container">
+    <div class="sidebar-title">
+      {% if page.edition == 'enterprise' %}{{site.ee_product_name}}
+      {% elsif page.edition == 'konnect' %}{{site.konnect_saas}}
+      {% elsif page.edition == 'mesh' %}{{site.mesh_product_name}}
+      {% elsif page.edition == 'deck' %}decK
+      {% elsif page.edition == 'kubernetes-ingress-controller' %}{{site.kic_product_name}}
+      {% elsif page.edition == 'gateway-oss' %}{{site.ce_product_name}}
+      {% elsif page.edition == 'getting-started-guide' %}Getting Started with Kong Gateway
+      {% endif %}
+    </div>
+    {% unless page.no_version == true %}
+    <div class="versions-dropdown dropdown">
```

Comment:
Can you adjust the version dropdown fit the width of the nav with the same padding as on the left?